manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/pipeline.yaml

```diff
@@ -149,6 +149,19 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
+      - pipelines.kubeflow.org
+    resources:
+      - scheduledworkflows
+      - workflows
+    verbs:
+      - report
+  - apiGroups:
+      - ''
+    resources:
+      - namespaces
```

Comment:
@zijianjoy Glad you catch the root cause of the failing tests! 
Without any prior knowledge of GCE and its tests, I want to mention that I see this "copy and paste" of config files as an error-prone place .. that could probably be automated. 
Glad that you catch the missing:
```
- apiGroups:
  - ''
  resources:
  - namespaces
  verbs:
  - get
 ```
It was actually missing for quite a lot of time.
Something that is still missing is: 
https://github.com/kubeflow/pipelines/blob/5b59e4a76fa4a086ebb4e6b590557bdd72e86cb5/manifests/kustomize/base/pipeline/ml-pipeline-persistenceagent-deployment.yaml#L26-L31

I guess that the GCE tests are run in multi-user mode = false, otherwise, I would expect `readArtifact` and `ReportRunMetric` to be failing. Is this the case?

Reply:
Thank you @difince for the information. Yes these GCE tests are in KFP standalone mode. I agree with you that we should automate the process of generating marketplace helm chart. 

In fact, when looking at the charts in gcp_marketplace folder, I sense that they are generated by kustomize at the first place. Unfortunately I cannot find the documented process for how to regenerate. We can consider introducing a kustomize overlay for marketplace, so most of future yaml changes can land on gcp_marketplace automatically.